server/util/post.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-twitter/server/constant"
+)
+
+func EphemeralPost(channelID, message string) *model.Post {
+	post := &model.Post{
+		ChannelId: channelID,
+		Message:   message,
+	}
+	post.SetProps(model.StringInterface{
```

Comment:
As far as I know, this is not needed. You can just put the UserID in the post as the bot id, and it will work the same way, right? Or is there any reason to do it this way?